recipes/c3/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "c3" %}
+{% set version = "0.4.18" %}
+{% set sha256 = "1521bf046745c347b6ebd3cfd1adb568355e702ffb85d272f3b6c2e82ec8b0ee" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/c3js/c3/archive/v0.4.18.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: True
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
+    
+test:
+  imports:
```

Comment:
Test that a file is installed.

Reply:
Thanks for the review. To use this package, we simply import the file(just like jquery) and I don't think any kind of import will work. Do you have any suggestion for testing?

Reply:
http://conda-forge.readthedocs.io/en/latest/testing.html#simple-existence-tests

Reply:
Thanks. I will look at it.